lib/pharos/phases/configure_master.rb

```diff
@@ -19,20 +19,24 @@ def install?
       end
 
       def call
-        push_kube_certs(cluster_context['master-certs']) if cluster_context['master-certs']
-
-        logger.info { "Checking if Kubernetes control plane is already initialized ..." }
-        if install?
-          logger.info { "Kubernetes control plane is not initialized." }
-          install
-          install_kubeconfig
-        elsif !cluster_context['api_upgraded']
-          reconfigure
-        else
-          logger.info { "Kubernetes control plane is up to date." }
-        end
+        Thread.current.abort_on_exception = true
```

Comment:
I don't like that Phase needs to configure these.

Reply:
I think it could safely be enabled for all phases.